build/Targets/Dependencies.props

```diff
@@ -101,4 +101,4 @@
     <xunitrunnerconsoleVersion>2.2.0-beta4-build3444</xunitrunnerconsoleVersion>
```

Comment:
Won't fix at this point. The package change we're making is known to have the same contents otherwise.